.idea/workspace.xml

```diff
@@ -2,20 +2,22 @@
 <project version="4">
```

Comment:
add .idea folder to .gitignore because this is not part of application code
see https://medium.com/@haydar_ai/learning-how-to-git-ignoring-files-and-folders-using-gitignore-177556afdbe3